code/modules/ai/ai_behaviors/human_mobs/human_mob.dm

```diff
@@ -38,6 +26,64 @@
 	COOLDOWN_DECLARE(ai_heal_after_dam_cooldown)
 	///Cooldown on retreating, so we don't get stuck running forever if pursued
 	COOLDOWN_DECLARE(ai_retreat_cooldown)
+	/// Cooldown on chat lines, to reduce spam
+	COOLDOWN_DECLARE(global_chat_cooldown)
+	/// Cooldown for specific chat lines, if applicable
+	var/list/specific_chat_cooldowns
+	/// Chat lines when moving to a new target
+	var/list/new_move_lines = list(
+		FACTION_NEUTRAL = list(
+			"Watch your spacing!",
+			"Changing position!",
+			"I'm on the way!",
+			"I'm hoofin' it!",
+			"Go, go, go!",
+			"Moving out!",
+			"Let's move.",
+			"Moving!",
+		),
+	)
+	/// Chat lins when receiving an order
+	var/list/receive_order_lines = list(
+		FACTION_NEUTRAL = list(
+			"Affirmative.",
+			"You got it.",
+			"Understood.",
+			"Right away.",
+			"Of course.",
+			"Roger.",
+			"Okay.",
+		),
+		FACTION_TERRAGOV = list(
+			"Who put you in charge?",
+			"I have it sorted.",
+			"On the double.",
+			"Affirmative.",
+			"Roger.",
+		),
+	)
+	/// Chat lines when following a new target
+	var/list/new_follow_chat = list(
+		FACTION_NEUTRAL = list(
+			"Yeah, I'll follow you.",
+			"I got your back!",
+			"Following you.",
+			"In formation!",
+			"Where to?",
+		),
+	)
+	/// Chat lines for retreating on low health
+	var/list/retreating_lines = list(
+		FACTION_NEUTRAL = list(
+			"RUN FOR YOUR LIFE!!",
+			"Cover me, I'm hit!",
+			"I need help here!",
+			"Falling back!",
+			"Disengaging!",
+			"HELP!!",
+			"RUN!!",
+		),
+	)
```

Comment:
as with all the rest, you're setting these as generic and organised by faction, in which case why are they at the datum level?